File: rest.js
const request = require('request-promise-native')
/**
 * if request(...) => return raw JSON string (use json.loads in Python)
 * if request(..., { json: true }) => return JS object (use demjson.decode in Python)
 */
const search = async name => request(`https://www.thecocktaildb.com/api/json/v1/1/search.php?s=${name}`)
module.exports = { search }

File: rest.py
import json
import requests

class Rest:
  def search(self, name):
    """ 
    if requests.get(...).text => return raw JSON string (use JSON.parse in JS)
    if requests.get(...).json() => return Python dictionary (need more complex parsing in JS)
    """
    return requests.get('https://www.thecocktaildb.com/api/json/v1/1/search.php?s=' + name).text
